### PR TITLE
Add documentation on how to convert RCAN model

### PR DESCRIPTION
**Pull request by achetver**
### Details:
 - Add instruction to convert [RCAN](https://github.com/yulunzhang/RCAN) model from PyTorch to ONNX format

### Tickets:
 - 39052

### CODE REVIEW

**Comment by achetver** [comment]
@mvafin @sadolini @yekruglov Please, review this PR

**Review by mvafin — APPROVED** [review]

**Review by yekruglov — APPROVED** [review]

**Comment by achetver** [comment]
@openvino-docs-maintainers, please, review this PR

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_RCAN.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from RCAN_TestCode.code.model.rcan import RCAN
+
+config = Namespace(n_feats=64, n_resblocks=4, n_resgroups=2, reduction=16, scale=[2], data_train='DIV2K', res_scale=1,
+                   n_colors=3, rgb_range=255)
+net = RCAN(config)
+net.eval()
+dummy_input = torch.randn(1, 3, 360, 640)
+torch.onnx.export(net, dummy_input, 'RCAN.onnx')
+```
+The script generates the ONNX\* model file RCAN.onnx. You can find more information about model parameters (`n_resblocks`, `n_resgroups` and others) in model repository and can use different values of them. The model conversion was tested with the repository hash commit `3339ebc59519c3bb2b5719b87dd36515ec7f3ba7`.
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_RCAN.md:25]:
```suggestion
The script generates the ONNX\* model file RCAN.onnx. You can find more information about model parameters (`n_resblocks`, `n_resgroups`, and others) in the model repository and use different values of them. The model conversion was tested with the repository hash commit `3339ebc59519c3bb2b5719b87dd36515ec7f3ba7`.
```

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ Here is the list of models that are tested and guaranteed to be supported. Howev
   steps 2 and 3 of [regular instructions](#typical-pytorch).
 * YOLACT topology can be converted using steps described in [Convert PyTorch\* YOLACT to the IR](pytorch_specific/Convert_YOLACT.md)
   instruction which is used instead of steps 2 and 3 of [regular instructions](#typical-pytorch).
+* [RCAN](https://github.com/yulunzhang/RCAN) topology can be converted using steps described in [Convert PyTorch\* RCAN to the IR](pytorch_specific/Convert_RCAN.md)
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md:24]:
```suggestion
* [RCAN](https://github.com/yulunzhang/RCAN) topology can be converted using the steps described in [Convert PyTorch\* RCAN to the IR](pytorch_specific/Convert_RCAN.md)
```

**Review by sadolini — APPROVED** [review]

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_RCAN.md (hunk 1 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_RCAN.md]: Done.

File: docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md (hunk 2 above)
Comment (achetver) [reply docs/MO_DG/prepare_model/convert_model/Convert_Model_From_PyTorch.md]: Done.

**Review by tsavina — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**